Makefile

```diff
@@ -341,7 +341,8 @@ androidupdate: all
 	# make android update apk
 	cd $(INSTALL_DIR)/koreader && 7z a -l -m0=lzma2 -mx=1 \
 		../../$(ANDROID_LAUNCHER_DIR)/assets/module/koreader-$(VERSION).7z * \
-		-x!resources/fonts -x!resources/icons/src -x!spec
+		-x!resources/fonts -x!resources/icons/src -x!spec \
+		-x!fonts/droid
```

Comment:
Similarly, the deb package should skip Droid (?) and Noto. `fonts-noto-core` can be a dependency and boom, half size package.

Reply:
In that case we need to remove the "enable system fonts" toggle.

Reply:
@pazos : Err, I'm not sure I follow? (Or did you mean we need to enforce reading system fonts in that case?)

Reply:
> we need to enforce reading system fonts in that case?

Exactly.